server/controllers/availability.js

```diff
@@ -7,13 +7,20 @@ const mongoose = require('mongoose')
 
   exports.updateAvailability = asyncHandler(async (req, res) => {
 
-     for(let prop in req.body.values.availability) if(!req.body.values.availability[prop]) delete req.body.values.availability[prop];//it will remove fields who are undefined or null
+     for(let prop in req.body.values.availability){
```

Comment:
Please make sure we are adding whitespace on our code `for (let props in req.body.values.availability) { . . .`

Also, nit: we could destructure values from the req.body object and access via `values.availability`

Reply:
OK